editoast/editoast_schemas/src/paced_train.rs

```diff
@@ -24,4 +51,120 @@ pub struct PacedTrain {
     pub train_schedule_base: TrainSchedule,
     #[schema(inline)]
     pub paced: Paced,
+    #[serde(default)]
+    #[schema(required)]
+    pub exceptions: Vec<PacedTrainException>,
+}
+
+/// Represents an exception for a paced train occurrence.
+/// Occurrences are normally generated from a base paced train.
+/// An exception occurs when an occurrence is added or modified
+/// compared to the base model. Each field corresponds to a "ChangeGroup" of attributes
+/// that may deviate from the paced train.
+/// - Created: A new occurrence manually added by the user, not originally present in the automatically generated occurrences based on the paced train.
+/// - Modified: An existing occurrence that has been changed
+#[derive(Debug, Clone, Default, Serialize, Deserialize, PartialEq, ToSchema)]
+pub struct PacedTrainException {
+    /// Unique key for the exception within the paced train, required and generated by the frontend.
+    pub key: String,
+    #[serde(flatten, default)]
+    #[schema(inline)]
+    pub exception_type: PacedTrainExceptionType,
```

Comment:
```suggestion
    #[serde(flatten)]
    #[schema(inline)]
    pub exception_type: PacedTrainExceptionType,
```

shouldn't Created {} be selected automatically when there's no field?